added praedikate section

jonasfabian opened the pull request.

Commit:
added praedikate section

FinalAngel commented on src/praedikate.tex:
```diff
@@ -1,3 +1,47 @@
+
+\textbf{Was sind Praedikate?}
+\\
+\\Bsp. Aussage über ein mathematisches Objekt: A≔5 ist eine ungerade Zahl
```
$A=5$, die Mathematischen Formeln immer zwischen $$ packen

FinalAngel replied:
Das gilt auch für alle weiteren Formeln

FinalAngel commented on src/praedikate.tex:
```diff
@@ -1,3 +1,47 @@
+\\
+\\Eine Aussage bestand bisher immer daraus, etwas über die Eigenschaft eines Objekts zu sagen. Solche Aussagen werden in der Mathematik «Praedikate» genannt.
+\\
+\\Die Zuordnung eines Praedikats wird wie folgt dargestellt: P(x). P ist das Praedikat, wobei x das Objekt ist.
```
Du kannst äöü verwenden :)

FinalAngel commented on src/praedikate.tex:
```diff
@@ -1,3 +1,47 @@
+\\
+\\Enthaelt ein Praedikat mehrere Objektvariablen, so nennen wir unser Praedikat «n-stelliges Praedikat» oder eine «n-stellige Aussageform». Wenn jedoch eine Aussage keine Variable enthaelt, reden wir von einem «0-stelligen Praedikat».
+\\
+\\Bsp.1: n-stelliges Praedikat: Q(x,\ y) := «x ist Automarke und y ist das Modell». Das Praedikat Q(VW,\ Golf) ist wahr, wobei Q(Toyota,\ Polo) falsch ist.
```
Am besten packst du Beispiele in eine Box:

```
\begin{align}
\fbox{\begin{minipage}{11.8cm}
  \textbf{Beispiel}: n-stelliges Prädikat: $Q(x,\ y) :=$ «x ist Automarke und y ist das Modell». Das Prädikat $Q(VW,\ Golf)$ ist wahr, wobei $Q(Toyota,\ Polo)$ falsch ist.
\end{minipage}}
\end{align}
```

FinalAngel commented on src/praedikate.tex:
```diff
@@ -1,3 +1,47 @@
+\\
+\\\textbf{Aequivalenz von zwei Praedikaten}
+\\
+\\Wenn alle Objektvariablenbelegungen dieselben Wahrheitswerte haben, dann wird das folgendermassen dargestellt: P(x) <==> Q(x) oder P(x) ==> Q(x) und Q(x) ==> P(x)
```
Die richtigen Latex symboloe verwenden:

<==> > `\iff`
==> > `\Rightarrow`

Also

```
$P(x) \iff Q(x)$
```

FinalAngel commented on src/praedikate.tex:
```diff
@@ -1,3 +1,47 @@
+\\
+\\\textbf{Quantoren}
+\\
+\\Wenn P irgendein Praedikat ist und x eine Objektvariable einer bestimmten Objektklasse, dann gilt: «für alle x gilt P(x)» die wird folgendermassen geschrieben: Ax:P(x)
```
Am besten erstellst du hier eine Übersicht mit den symbolen zuerst, ähnlich wie bei mir:

```
\def\arraystretch{1.5}
\begin{table}[ht]
\begin{tabular}[t]{ll}
\hline
  Kürzel & Definition\\
\hline
  $A, B$ & Variabeln\\
  $\land$ & Konjunktion, logisches UND\\
  $\lor$ & Disjunktion, logisches ODER\\
  $\lnot$A oder $\bar{A}$ & Negation\\
  $\Rightarrow$ & Implikation ($A$ = Prämisse, $B$ = Konklusion)\\
  $\iff$ & Äquivalenz\\
\hline
\end{tabular}
\end{table}
```

Einfach mit den Quantoren, diese findest du auch hier:
https://de.wikipedia.org/wiki/Pr%C3%A4dikatenlogik

Du kannst bei WIkipedia einfach copy paste machen, dann bekommst du die Zeichne: `{\displaystyle \forall x({\text{Katze}}(x)\Rightarrow {\text{Säugetier}}(x))}`

FinalAngel commented on src/praedikate.tex:
```diff
@@ -1,3 +1,47 @@
+\\Wenn es jedoch genau nur ein Objekt gibt, dann: «es gibt genau ein x, so dass P(x) gilt». Dies wird wie folgt dargestellt: E!x:P(x).
+\\
+\\Das Symbol E! wird Eindeutigkeitsquantor oder Einzigkeitsquantor genannt.
+\\
```
Es fehlen noch weitere Beispiele, wie z.B. bei dem Wikipedia Artikel

FinalAngel commented on src/praedikate.tex:
```diff
@@ -1,3 +1,47 @@
+\\Das Symbol E! wird Eindeutigkeitsquantor oder Einzigkeitsquantor genannt.
+\\
+\\\textbf{Ausserdem: Ein Quantor bindet staerker als alle anderen logischen Operatoren!!!}
+\\
```
Es fehlt noch:

* Prädikatenlogischen Formeln sowie Beispiele
* Rechenregeln (Verneinungssäze / Vertauschbarkeitssätze)
* Rechenbeispiel, wie man dieses anwendet

Du kannst auch ruhik Sturkuren bei mir kopieren für die Rechenregeln (tabelle) etc.

FinalAngel commented:
Bitte füg auch noch einen entsprechenden Eintrag in die `src/_revisionen.tex` ein

Zer0SkiLL commented:
grundsätzliche formatierung (bei angelo abschauen). aber wenn die handschriftliche notiz reicht, sieht das auch wieder anderst aus, dann schulden wir angelo einfach 2 kisten bier ;)

jonasfabian committed:
-formulas are between dollar signs now
-added äöü
-put examples in boxes
-used Latex arrows
-added formula tables
-added additional examples to "Quantoren"
-added "Verneinungssäze / Vertauschbarkeitssätze"

Commit:
-added examples to Prädikate table
-added table for Verneinugs- and Vertauschungssätze
-added examples for those rules
-added entry in revision file
